PocketKit/Sources/PocketKit/Home/RecentSavesItemCell.swift

```diff
@@ -30,16 +30,21 @@ class RecentSavesItemCell: HomeCarouselItemCell {
             self.thumbnailURL = CDNURLBuilder().imageCacheURL(for: item.topImageURL)
         }
 
+        var attributedCollection: NSAttributedString? {
+            guard item.isCollection else { return nil }
+            return NSAttributedString(string: Localization.Collection.title, style: .recommendation.collection)
+        }
+
         var attributedTitle: NSAttributedString {
-            NSAttributedString(string: title, style: .title)
+            NSAttributedString(string: title, style: .recommendation.title)
         }
 
         var attributedDomain: NSAttributedString {
-            return NSAttributedString(string: domain ?? "", style: .domain)
+            return NSAttributedString(string: domain ?? "", style: .recommendation.domain)
```

Comment:
Will we ever care enough about nil domain or timeToRead enough to log something?

Reply:
this was already pre-existing and the change only involves updating how the style is accessed, but I would say I have seem items that do not have `timeToRead` such as collections so that's okay. In terms of domain, if we do want to log, I think we should do it at the item level as opposed to this location.